homeassistant/components/websocket_api/commands.py

```diff
@@ -310,6 +319,69 @@ async def handle_integration_setup_info(
     )
 
 
+@decorators.websocket_command({vol.Required("type"): "integration/log_info"})
+@decorators.async_response
+async def handle_integration_log_info(
+    hass: HomeAssistant, connection: ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Handle integrations logger info."""
+    connection.send_result(
+        msg["id"],
+        [
+            {
+                "domain": integration,
+                "level": logging.getLogger(
```

Comment:
Instead of this, I think that we could just return the integrations that are currently enabled.